ios/RNSScreenStackHeaderSubview.mm

```diff
@@ -0,0 +1,132 @@
+#import "RNSScreenStackHeaderSubview.h"
+#import "RNSConvert.h"
+
+#ifdef RN_FABRIC_ENABLED
+#import <react/renderer/components/rnscreens/ComponentDescriptors.h>
+#import <react/renderer/components/rnscreens/EventEmitters.h>
+#import <react/renderer/components/rnscreens/RCTComponentViewHelpers.h>
+
+#import <React/RCTConversions.h>
+#import "RCTFabricComponentsPlugins.h"
+#endif
+
+@implementation RNSScreenStackHeaderSubview {
+#ifdef RN_FABRIC_ENABLED
+  BOOL _isInitialValueSet;
+#endif
+}
+
+#pragma mark - Common
+
+#ifdef RN_FABRIC_ENABLED
+
+#pragma mark - Fabric specific
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+  if (self = [super initWithFrame:frame]) {
+    static const auto defaultProps = std::make_shared<const facebook::react::RNSScreenStackHeaderSubviewProps>();
+    _props = defaultProps;
+  }
+
+  return self;
+}
+
+#pragma mark - RCTComponentViewProtocol
+
+- (void)prepareForRecycle
+{
+  [super prepareForRecycle];
+  _isInitialValueSet = NO;
```

Comment:
Looks like this prop is not used anywhere

Reply:
It's hard to tell if it is just a leftover it was intended for future use, but right now it is useless indeed. Removing it: [4ca37ef](https://github.com/software-mansion/react-native-screens/pull/1418/commits/4ca37ef5e616be7c21f243ed3f370cc08a474ae9)